cmd/sandbox/start.go

```diff
@@ -191,7 +191,7 @@ func startSandbox(ctx context.Context, cli docker.Docker, reader io.Reader) (*bu
 	}
 	sandboxImage := sandboxConfig.DefaultConfig.Image
 	if len(sandboxImage) == 0 {
-		image, version, err := githubutil.GetFullyQualifiedImageName(sandboxConfig.DefaultConfig.Version, sandboxImageName, sandboxConfig.DefaultConfig.Prerelease)
+		image, version, err := githubutil.GetFullyQualifiedImageName("dind", sandboxConfig.DefaultConfig.Version, sandboxImageName, sandboxConfig.DefaultConfig.Prerelease)
```

Comment:
Instead of passing it can we add this to the config .
ie have the 
```
var (
	DefaultSingleBinaryConfig = &Config{
   prefix : "sha"
}
DefaultNonSingleBinaryConfig = &Config{
   prefix : "dind"
}
)

```
type Config struct {
...
	Prefix  string `json:"prefix" pflag:-", "` // dont generate pflag for this
}
```

Reply:
that is not needed because it's the past debt we have on our sandbox image, We never user non dind image anywhere, we want to move on sha prefix for sandbox. 

we have `--image` flag for specifying the image full name